workflow/rocoto/tasks.py

```diff
@@ -176,6 +177,13 @@ def get_resource(self, task_name):
 
         task_config = self._configs[task_name]
 
+       #The PW_CSP is a AWS (CSPs parameter), if it is on CSPs,
+       #use "$USER" as account.
+        pw_csp = os.environ.get('PW_CSP', None)
+        if ( pw_csp in ['aws', 'azure', 'google'] ):
+            task_config['ACCOUNT'] = os.environ.get('USER')
+            task_config['ACCOUNT_SERVICE'] = os.environ.get('USER')
```

Comment:
AWSPW.yaml has these set to `USER`, so why is this needed here?

Reply:
Removed. At first, did not figure out this. Thanks for point out.